test_track_rails_client.gemspec

```
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'faraday_middleware'
   s.add_dependency 'mixpanel-ruby', '~> 1.4'
   s.add_dependency 'multi_json', '~> 1.7'
-  s.add_dependency 'public_suffix', '>= 2.0.0', '<= 3.0.0'
+  s.add_dependency 'public_suffix', '~> 4.0.6'
```

I think we want to keep the same minimum version as before, so that consuming apps still have the option to decide.

```suggestion
  s.add_dependency 'public_suffix', '>= 2.0.0'
```